pages/common/xml-escape.md

```diff
@@ -0,0 +1,16 @@
+# xml escape
+
+> Escape special XML characters, e.g. `<a1>` -> `&lt;a1&gt;`.
```

Comment:
Since all clients support UTF8, what do you think about
```suggestion
> Escape special XML characters, e.g. `<a1>` → `&lt;a1&gt;`.
```

Reply:
Yes.